### PR TITLE
doc: rewrite of the readme

### PR DESCRIPTION
**Pull request by mpvader**
See commits for what has changed; and why.

### CODE REVIEW

**Comment by mpvader** [comment]
there is some issue with a label, but I don't see how I can label the PR.

**Review by cmotelet — COMMENTED** [review]
Maybe also add an entry with the link to the Windows installer
https://github.com/SignalK/signalk-server-windows

**Review by tkurki — COMMENTED** [review]
The diff is all over the place, but I'll put this here: let's not leave docker out. This is actually something that I often look for myself when looking into some new to me software: how do I get it running locally via docker with a single copypaste command?

And 👍 on adding an entry for windows.

**Comment by mpvader** [comment]
Hey both, thanks for the review. I didn’t even know there was a windows installer.

Please let me know if ok like this and if you see any more improvements.

**Comment by mpvader** [comment]
Ps. I’m a bit new at pull requests, the idea of those extra two commits is to have them like this so you can see clearly what I changed; and then later to squash them into the main rewrite-commit.

**Comment by cmotelet** [comment]
In the Windows readme.md I have described the very first operation to do after installing the server.
The creation of the  `admin` user.
I think this important point should be added a little earlier in the documentation in the `configuration-and-use` section
rather than in `Restarting Signal K Server` section.
https://github.com/SignalK/signalk-server-windows/blame/master/readme.md#L76-L80

**Comment by mpvader** [comment]
Hi again, done.

**Review by tkurki — CHANGES_REQUESTED** [review]
Changes/comments based on ToC:
- **Introduction** NICE! change _Software Developers_ to _Software Developers & Boat Electronics Hobbyists_? there's a community of SensESP people. Add a link to SensESP and ESP32 at/near sensor mention
- **Signal K Platform**
- **Community & support**  Move link to FAQ here, drop FAQ heading
- **How to get Signal K Server** installing from NPM should just one more option here under this heading. Change the text about Docker to something like _You can run the server in Docker, see [Docker Quickstart](https://github.com/SignalK/signalk-server/blob/master/docker/README.md#quickstart)_. Drop the provisioning option.
- **Other necessary hardware** drop this heading, add a link to NMEA 2000 near the FAQ link and/or make the nmea2000 mentions links to FAQ.
- **Installing from NPM** drop this heading, see above
- **Configuration and use** VERY NICE
- **FAQ** drop this heading, see above
- **Supported PGNs, Sentences and more**
- **Environment variables**  move to the end, before 
- **Settings files** this is out of date in many ways, just remove the whole damn thing
- **Debugging** maybe drop this section and add a note about Server Log in the configuration section? or this text and a screenshot?
- **Development** drop the releasing.md link, not relevant for target audience
- **Sponsoring Signal K** 
- **License** keep but also add badge `GitHub` in the beginning

**Comment by tkurki** [comment]
..and yes, just push more commits, we can squash easily when merging.

**Comment by mpvader** [comment]
Hi,

Thank you for the extensive review. All done.

Pls make sure to check my last commit, `elaborate on env variable SIGNALK_NODE_CONFIG_DIR`, extra careful. Going by [the start script in Venus OS](https://github.com/victronenergy/meta-victronenergy/blob/master/meta-third-party/recipes-extended/signalk-server/signalk-server/start-signalk.sh), thats how must work, but I'm not 100% sure.

**Comment by mpvader** [comment]
And, after looking around a bit more, shall I add these to the list of available command line options?

- `--sample-nmea0183-data`: starts signalk-server with sample NMEA0183 data.
- `--sample-n2k-data`: starts signalk-server with sample NMEA2000 data.
- `--override-timestamps`: overrides timestamps in the sample data with current date and time.
- `--securityenabled`: one of the ways to enable security. For a fresh install this means that the Admin UI forces the user to create an admin account before he/she can continue into the UI. See [SECURITY.md#enabling-security](https://github.com/SignalK/signalk-server/blob/master/SECURITY.md#enabling-security) for further details.

File: README.md [hunk 1]
```diff
@@ -220,7 +220,7 @@ To enable debugging without going through the Admin UI, see the file `~/.signalk
 - `ADMINUSER` force a account for admin user (username:password format).
 - `PRESERIALCOMMAND` command to run before opening a serial port.
 - `SIGNALK_NODE_SETTINGS` override the path to the settings file.
-- `SIGNALK_NODE_CONFIG_DIR` override the path to find server configuration files.
+- `SIGNALK_NODE_CONFIG_DIR` override the path to find server configuration. Includes all run-time changing content: configuration files, plugins, plugin configuration files, webapps, and so forth. Used when installing signalk-server on a read-only file system.
```
Comment (tkurki) [inline README.md]: "what's there" 👍 

I'd remove the read only file system comment - not relevant for 99.9% of the people.

**Comment by tkurki** [comment]
👍 on adding command line options

**Comment by mpvader** [comment]
👍

**Pull request merged**